edx_proctoring/apps.py

```diff
@@ -53,7 +53,7 @@ def make_worker_config(backends, out='/tmp/workers.json'):
         try:
             package = backend.npm_module
             package_file = os.path.join(settings.NODE_MODULES_ROOT, package, 'package.json')
-            with open(package_file, 'r') as package_fp:
+            with open(package_file, 'r', encoding='utf-8') as package_fp:
```

Comment:
Thank you for separating these into individual commits; I didn't realize immediately that there was only on line that needed to be fixed in this way. Separating them helped me see that much more easily.

The files this line can load are git-managed source code, so it's reasonably safe to assume they'll remain utf-8 decodable. In particular they're https://github.com/anupdhabarde/edx-proctoring-proctortrack/blob/master/package.json and https://github.com/edx/mockprock/blob/master/package.json.